src/routes/safe/components/Transactions/TxList/utils.ts

```diff
@@ -199,3 +199,7 @@ export const isDeeplinkedTx = (): boolean => {
 export const isAwaitingExecution = (
   txStatus: typeof LocalTransactionStatus[keyof typeof LocalTransactionStatus],
 ): boolean => [LocalTransactionStatus.AWAITING_EXECUTION, LocalTransactionStatus.PENDING_FAILED].includes(txStatus)
+
+export const camelCaseToSpaces = (str: string): string => {
```

Comment:
I don't think this is a `TxList` specific utility. I'd move it to a more generic place.

Reply:
Moved to `src/utils/camelCaseToSpaces.ts`